acceptance_tests/features/steps/supporttool_ui.py

```diff
@@ -181,21 +187,21 @@ def creating_email_template(context, packcode, template: List):
 
 
 @step('I should see the email template in the template list')
-def finding_created_email_template(context):
+def find_created_email_template(context):
     test_helper.assertEquals(
         len(context.browser.find_by_id('emailTemplateTable').first.find_by_text(context.pack_code)), 1)
 
 
 @step("the email template has been added to the allow on action rule list")
-def allowing_email_template_on_action_rule(context):
+def allow_email_template_on_action_rule(context):
     context.browser.find_by_id('allowEmailTemplateDialogBtn').click()
     context.browser.find_by_id('selectEmailTemplate').click()
-    context.browser.find_by_value(context.pack_code).click()
+    context.browser.find_by_id(context.pack_code).click()
```

Comment:
This is the main bug fix. When using `find_by_value` it complains that the click would be intercepted by another component, a div which is `aria-hidden=true` which I believe is for greying out the background when the modal is shown. It's part of the MUI components, not our own code and it doesn't cause problems anywhere else, even on this same modal/find method when there is more than one template list item to choose from. `find_by_id` doesn't seem to hit the same issue.